pixeltable/env.py

```diff
@@ -423,6 +416,16 @@ def _set_up(self, echo: bool = False, reinit_db: bool = False) -> None:
                 f'(either add a `file_cache_size_g` entry to the `pixeltable` section of {Config.get().config_file},\n'
                 'or set the PIXELTABLE_FILE_CACHE_SIZE_G environment variable)'
             )
+
+        self._default_input_media_dest = config.get_string_value('input_media_dest')
```

Comment:
Missing from class-level decls and init().

Reply:
Fixed in class decls.

Do we actually need to set them in `init()`? I'd argue it's better to get an error if an attempt is made to access them before calling `set_up()`, rather than silently (and often incorrectly) defaulting to `None`.

I guess we could load them in config during `init()` instead of `set_up()`.